tests/preview/test_api.py

```diff
@@ -0,0 +1,24 @@
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+
+# TODO(vmttn): find actual data to use for these tests
```

Comment:
did you want to do this in this PR ?

Reply:
I think it would be best to do this now, but it's not something new (`preview` and `delete-resources` endpoints) and I am not sure about the data required for this request.